includes/widgets/icon-box.php

```diff
@@ -740,4 +737,4 @@ protected function content_template() {
 	public function on_import( $element ) {
 		return Icons_Manager::on_import_migration( $element, 'icon', 'selected_icon', true );
 	}
-}
+}
```

Comment:
Add an empty line at the end of this file please

Reply:
added